server/players.py

```diff
@@ -173,7 +173,7 @@ def __repr__(self) -> str:
                 f"clan={self.clan}, game_count={dict(self.game_count)})")
 
     def __hash__(self) -> int:
-        return self.id
+        return hash(self.id)
```

Comment:
This should produce more uniformly distributed values. It probably doesn't matter too much in practice since none of our dictionaries that use players as keys should be getting anywhere close to being as large as the total player base, but in theory at least this should give general better dict performance and probably prevent some DOS attacks as well.